app/celery/tasks.py

```diff
@@ -159,6 +159,7 @@ def process_rows(rows: List, template: Template, job: Job, service: Service):
         signed_row = signer.sign(
             {
                 "api_key": job.api_key_id and str(job.api_key_id),
+                "key_type": job.api_key.key_type if job.api_key else KEY_TYPE_NORMAL,
```

Comment:
I'm not sure if it's possible for `job.api_key` not to exist but I added a check here so that if it didn't it would default to normal as it does elsewhere in the system.  If this should default more strictly (i.e. to a TEST type), please let me know.